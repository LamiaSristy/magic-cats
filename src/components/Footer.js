import React from 'react';
import '../stylesheets/Footer.scss';

const Footer = () => (
  <footer className="footer">
    <div className="container">
      Copyright &copy; All rights reserved - by Lamia$risty
    </div>
  </footer>
);

export default Footer;
