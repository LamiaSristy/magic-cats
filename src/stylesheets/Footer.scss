@import '~bootstrap/scss/bootstrap.scss';

.container {
  max-width: 1500px;
  margin: 0 auto;
}

.footer {
  font-family: Arial, Helvetica, sans-serif;
  background-color: #303233;
  color: #fff;
  padding: 0 5px;
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
  box-shadow: 0 -0.05rem 0.5rem  #303233;
  position: fixed;
  left: 0;
  right: 0;
  bottom: 0;
  height: 60px;
  width: 100%;
}
