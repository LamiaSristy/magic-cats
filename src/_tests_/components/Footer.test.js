import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

const setup = () => shallow(<Footer />);

describe('Footer component', () => {
  let component;

  beforeEach(() => {
    component = setup();
  });

  it('Should render', () => {
    const element = component.find('.footer');
    expect(element.length).toEqual(1);
  });
});
