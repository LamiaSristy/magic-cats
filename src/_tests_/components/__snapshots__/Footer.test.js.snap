// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the cat-images the page 1`] = `ShallowWrapper {}`;

exports[`Footer component renders the footer 1`] = `ShallowWrapper {}`;
